talkideas.md

```diff
@@ -8,3 +8,39 @@ In-depth debugger talk
 ======================
 *TODO: Rewrite*
 
+
+Developing Developers
+=====================
+Software development is a young profession. Our career paths are still geared up to accept junior developers from academia, make them productive in a team, and then preparing them for a career in management. Technical skills and experience are devalued in preference of line management and project management.
+
+We have a responsibility for developing the technical skills and experience of developers, for building a better profession and for creating a healthy environment to work in. How can we shape careers that challenge and value developers, and how do you identify and respond when things go wrong?
+
+This talk will use my insights from building career paths from scratch at emerging companies, and examine the successes and failures that are common across the industry.
+
+
+Modular CSS with WebPack
+========================
+Composition has long been a goal of CSS design. Creating a rich library of styles that can be reused and combined saves time in the long run, but often lead to unmaintainable or verbose code.
+
+WebPack is commonly used for bundling JS modules, but its flexibility means it can also be used to create modular stylesheets: local styles that can be imported and exported just like scripts.
+
+This talk introduces WebPack and will demonstrate how to write, bundle and use CSS modules. We'll also examine the benefits of using WebPack, and why it can help in ways other tools have failed in the past.
+
```

Comment:
Is this CSS modules the project (it sounds really like it), or just modules/components that are CSS?

Otherwise it looks great. :) I'd really like to watch this talk, especially as I'm speaking about CSS modules in places. :D